### DISCUSSION TITLE
No -stable releases since v1.44.23-stable

### DISCUSSION THREAD

**Discussion by stronk7** [BerriAI/litellm#5870] [Q&A] [answered]
Hi,

I've seen that, since [v1.44.23-stable](https://github.com/BerriAI/litellm/releases/tag/v1.44.23-stable), there aren't stable releases any more (and we are, right now, on [v1.48.0](https://github.com/BerriAI/litellm/releases/tag/v1.48.0).

Before asking, I've tried to look in GH workflows to see if there were any job failing and causing that, but I've been unable to find anything about how/when those stable releases are made (I've read in various places in the docs that those are the ones to use, because they come with more testing and so on).

So this is just a question about the lack of them, lately. Is it normal (because there is something failing that prevents them to be released), or maybe the releases naming schema has changed and the docs are outdated?

TIA and ciao :-)

**Comment by ishaan-jaff** [comment, marked as answer]
hi @stronk7 - hoping to have a stable release out this week. 

Why has it not been out since 1.44.23 ? 

- We saw a ~50 - 100 ms perf increase on load tests, investigating this and hoping to have a stable release out soon
